routes/APIcall.js

```diff
@@ -1,17 +1,55 @@
 var express = require('express');
+const axios = require('axios');
+
 const router = express.Router();
 
-const axios = require('axios');
 
 router.get("/", function(req,res) {
-    axios.get("https://api.openweathermap.org/data/2.5/weather?q=Seattle&appid="+ process.env.WEATHER_API_KEY)
-      .then(function(response) {
-          res.send(response.data);
-      })
-      //catch error
-      .catch(function(error) {
-          res.sendStatus(400);
-      })
+  axios.get("https://api.openweathermap.org/data/2.5/weather?q=Seattle&appid=" + process.env.WEATHER_API_KEY)
```

Comment:
is this being used anymore?

Reply:
No, although ultimately a route like this will exist when we have the search bar implemented (for getting weather data by city/town name instead of by coordinates)